proto/proto/msgs.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+option go_package = "github.com/summa-tx/relays/proto";
+
+package relayproto;
+
+import "shared.proto";
+
+// MsgIngestHeaderChain defines a IngestHeaderChain message
+message MsgIngestHeaderChain {
+	string Signer = 1;
+	repeated BitcoinHeader Headers = 2;
+}
+
+// MsgIngestDifficultyChange defines a IngestDifficultyChange message
+message MsgIngestDifficultyChange {
+  string Signer = 1;
+  bytes Start = 2;
+  repeated BitcoinHeader Headers = 3;
+}
+
+// MsgMarkNewHeaviest defines a MarkNewHeaviest message
+message MsgMarkNewHeaviest {
+  string Signer = 1;
```

Comment:
protobuf names should be in `snake_case`. It will automatically rename to `CapsCase` when it generates the golang code.

```
    string signer = 1;
    bytes ancestor = 2;
    bytes current_best = 3;
    bytes new_best = 4;
    uint32 limit = 5;
```